nemo_text_processing/text_normalization/rw/taggers/time.py

```diff
@@ -0,0 +1,106 @@
+            ('06', " n'iminota itandatu") ,
+            ('07', " n'iminota irindwi") ,
+            ('08', " n'iminota umunani") ,
+            ('09', " n'iminota icyenda") ,
+            ('10', " n'iminota icumi") ,
+            ('11', " n'iminota cumi n'umwe") ,
+            ('12', " n'iminota cumi n'ibiri") ,
+            ('13', " n'iminota cumi n'itatu") ,
+            ('14', " n'iminota cumi n'ine") ,
+            ('15', " n'iminota cumi n'itanu") ,
+            ('16', " n'iminota cumi n'itandatu") ,
+            ('17', " n'iminota cumi n'irindwi") ,
+            ('18', " n'iminota cumi n'umunani") ,
+            ('19', " n'iminota cumi n'icyenda") ,
+            ('20', " n'iminota makumyabiri") ,
+            ('21', " n'iminota makumyabiri na rimwe") ,
+            ('22', " n'iminota makumyabiri n'ibiri") ,
+            ('23', " n'iminota makumyabiri n'itatu") ,
+            ('24', " n'iminota makumyabiri n'ine") ,
+            ('25', " n'iminota makumyabiri n'itanu") ,
+            ('26', " n'iminota makumyabiri n'itandatu") ,
+            ('27', " n'iminota makumyabiri n'irindwi") ,
+            ('28', " n'iminota makumyabiri n'umunani") ,
+            ('29', " n'iminota makumyabiri n'icyenda") ,
+            ('30', " n'iminota mirongo itatu") ,
+            ('31', " n'iminota mirongo itatu n'umwe") ,
+            ('32', " n'iminota mirongo itatu n'ibiri") ,
+            ('33', " n'iminota mirongo itatu n'itatu") ,
+            ('34', " n'iminota mirongo itatu n'ine") ,
+            ('35', " n'iminota mirongo itatu n'itanu") ,
+            ('36', " n'iminota mirongo itatu n'itandatu") ,
+            ('37', " n'iminota mirongo itatu n'irindwi") ,
+            ('38', " n'iminota mirongo itatu n'umunani") ,
+            ('39', " n'iminota mirongo itatu n'icyenda") ,
+            ('40', " n'iminota mirongo ine") ,
+            ('41', " n'iminota mirongo ine n'umwe") ,
+            ('42', " n'iminota mirongo ine n'ibiri") ,
+            ('43', " n'iminota mirongo ine n'itatu") ,
+            ('44', " n'iminota mirongo ine n'ine") ,
+            ('45', " n'iminota mirongo ine n'itanu") ,
+            ('46', " n'iminota mirongo ine n'itandatu") ,
+            ('47', " n'iminota mirongo ine n'irindwi") ,
+            ('48', " n'iminota mirongo ine n'umunani") ,
+            ('49', " n'iminota mirongo ine n'icyenda") ,
+            ('50', " n'iminota mirongo itanu") ,
+            ('51', " n'iminota mirongo itanu n'umwe") ,
+            ('52', " n'iminota mirongo itanu n'ibiri") ,
+            ('53', " n'iminota mirongo itanu n'itatu") ,
+            ('54', " n'iminota mirongo itanu n'ine") ,
+            ('55', " n'iminota mirongo itanu n'itanu") ,
+            ('56', " n'iminota mirongo itanu n'itandatu") ,
+            ('57', " n'iminota mirongo itanu n'irindwi") ,
+            ('58', " n'iminota mirongo itanu n'umunani") ,
+            ('59', " n'iminota mirongo itanu n'icyenda") ,
+        ])
+        
```

Comment:
Are there any formations that are similar to (using English examples) `quarter 'til five` `half past one`? May want to include to improve the expressive power of the graph.

Reply:
There is no direct similarity, but you can say "n'igice" (to mean "a half") or "na mirongo itatu" for minutes. Or you can say, " Ni saa tanu ibura iminota irindwi. " to say " It is seven to eleven. " or  "Ni saa tanu irenga iminota makumyabiri" to say  "It is twenty past eleven. "  but these can be said for any combination of hour and minutes if that makes sense.

Reply:
Hmm, if its a regular construction, lets add support for that. So it just duplicates this string map but reverses it so it gives 60-minutes. Should help make the graph more robust.